<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ReceiveBox_Window</class>
 <widget class="QDialog" name="ReceiveBox_Window">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1344</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>收件箱</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QTextBrowser" name="view_information">
   <property name="geometry">
    <rect>
     <x>550</x>
     <y>150</y>
     <width>751</width>
     <height>361</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="view_list">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>160</y>
     <width>401</width>
     <height>351</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_forwardthis">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>520</y>
     <width>231</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>转发该邮件</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_delete">
   <property name="geometry">
    <rect>
     <x>990</x>
     <y>520</y>
     <width>191</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>删除</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>650</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="text">
    <string>附件名称：</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_download">
   <property name="geometry">
    <rect>
     <x>1100</x>
     <y>690</y>
     <width>100</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>下载附件</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>690</x>
     <y>700</y>
     <width>101</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>请输入下载地址：</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="attadress">
   <property name="geometry">
    <rect>
     <x>790</x>
     <y>690</y>
     <width>291</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextBrowser" name="view_attname">
   <property name="geometry">
    <rect>
     <x>750</x>
     <y>640</y>
     <width>541</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1351</width>
     <height>761</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Background4.png</pixmap>
   </property>
   <property name="scaledContents">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>label_5</zorder>
  <zorder>label</zorder>
  <zorder>view_information</zorder>
  <zorder>view_list</zorder>
  <zorder>btn_forwardthis</zorder>
  <zorder>btn_delete</zorder>
  <zorder>label_3</zorder>
  <zorder>btn_download</zorder>
  <zorder>label_4</zorder>
  <zorder>attadress</zorder>
  <zorder>view_attname</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
